### DISCUSSION TITLE
./data, ./examples/data missing?

### DISCUSSION THREAD

**Discussion by pdurusau** [safevideo/autollm#167] [Q&A] [answered]
Am I just missing something really obvious? I cloned the project but don't have ./data or ./examples/data files/directories? Is there another location for those files?

**Comment by talhasarit** [comment, marked as answer]
Hi @pdurusau,

This 'examples/data' directory can be any folder that you have your own documents to ask questions about via `AutoQueryEngine`, if you want to use local files or a folder. It is just a placeholder :)

Please don't hesitate to ask if you need further assistance.
